heterocl/utils.py

```diff
@@ -133,8 +133,20 @@ def get_src_loc(frame=0):
 def make_const_tensor(val, dtype):
     # val is numpy ndarray
     if isinstance(dtype, (Int, UInt)):
-        if dtype.bits <= 64:
+        if dtype.bits == 1:
+            np_dtype = np.bool_
+        elif dtype.bits <= 8:
+            np_dtype = np.int8
+        elif dtype.bits <= 16:
+            np_dtype = np.int16
+        elif dtype.bits <= 32:
+            np_dtype = np.int32
+        elif dtype.bits <= 64:
             np_dtype = np.int64
+        elif dtype.bits <= 128:
+            np_dtype = np.int128
+        elif dtype.bits <= 256:
+            np_dtype = np.int256
```

Comment:
Could you add the same np_dtype set up for Fixed and UFixed cases? They are just like the Int and UInt cases, but the numpy array stores the value of `val * (2**dtype.fracs)`, e.g. for 2.25 in Fixed(5, 2) the stored value is 2.25x4=9 in  Int(5). It would also be good to add a corresponding test case for fixed-point types

Reply:
What do you mean? `Fixed` and `UFixed` cases have already been covered in this `make_const_tensor` function, and we also have test cases for fixed-point types.

Reply:
I meant these two cases here: https://github.com/cornell-zhang/heterocl/blob/2094c7c7c43792bc9c7b5366d41cd97f67f6a683/heterocl/utils.py#L151-L167
They are still using 64-bit